crates/modelardb_compression/src/models/mod.rs

```diff
@@ -348,6 +365,59 @@ mod tests {
         assert!(ErrorBound::try_new(f32::NAN).is_err())
     }
 
+    // is_value_within_error_bound().
+    proptest! {
+    #[test]
+    fn test_same_value_is_always_within_error_bound(value in ProptestValue::ANY) {
+        prop_assert!(is_value_within_error_bound(ErrorBound::try_new(0.0).unwrap(), value, value));
+    }
+
+    fn test_other_value_is_never_within_error_bound_of_positive_infinity(value in ProptestValue::ANY) {
```

Comment:
It seems that these tests are missing the `#[test]` header. I am not sure the tests are being run currently.

Reply:
Good catch, they are not. The tests are also wrong as I forgot to include `!` to specify that the special `f32` values should never be within the error bound of other values.